src/python/bot/fuzzers/syzkaller/engine.py

```diff
@@ -130,7 +132,8 @@ def reproduce(self, target_path, input_path, arguments, max_time):  # pylint: di
     syzkaller_runner = runner.get_runner(
         os.path.join(binary_dir, constants.SYZ_REPRO))
     repro_args = runner.get_config()
-    repro_args.extend(input_path)
+    repro_args.extend(
+        ['-infinite=false', '-restart_time={}s'.format(REPRO_TIME), input_path])
```

Comment:
As per Max's comment, infinite=false means restart time is the max run time, so i think restart_time argument does not apply. if you want to keep restart_time, i think you need to set infinite=true. i am fine with removing restart_time arg if we are only trying to reproduce it once (and wont have more time).

Reply:
From syzkaller:
	flagRestartTime = flag.Duration("restart_time", time.Hour, "how long to run the test")
	flagInfinite    = flag.Bool("infinite", true, "by default test is run for ever, -infinite=false to stop on crash")
Running under infinite=false and restart_time=120 means running until crash is found or till 2m0s (so possibly few tries). If you remove restart_time, it will only run once. I think we can afford to let it run few times, that s better than running once and failing to repro but maybe second or third could work.
Now infinite default value is true, if i remove it, and keep restart_time=120, this is the behavior:
2020/07/13 10:57:29 running infinitely and restarting VM every 2m0s
So even after a crash is found, it will still run.
2020/07/13 10:57:29 running infinitely and restarting VM every 2m0s
2020/07/13 10:57:29 reproducing from log file: /usr/local/google/home/hamzeh/HT75H1A00107/syzkaller/crashes/c684119dc4cf71e8d82ee041b591c3a09139a6cf/log0
2020/07/13 10:57:29 booting 1 test machines...
2020/07/13 10:57:29 vm-0: starting
2020/07/13 10:58:20 failed to associate adb device HT75H1A00107 with console: no console is associated with this device
2020/07/13 10:58:20 falling back to 'adb shell dmesg -w'
2020/07/13 10:58:20 note: some bugs may be detected as 'lost connection to test machine' with no kernel output
2020/07/13 10:58:21 device HT75H1A00107: battery level 98%, OK
2020/07/13 10:58:22 vm-0: crushing...
2020/07/13 10:58:32 vm-0: crash: panic:LINE, dump:LINE
2020/07/13 10:58:32 vm-0: starting
2020/07/13 10:58:32 saving crash panic:LINE, dump:LINE to /usr/local/google/home/hamzeh/HT75H1A00107/syzkaller/crashes/ea52b31f6e5d74acc123f0a3e7b8d77c021aa5bd
2020/07/13 10:58:32 instances executed: 1, crashes: 1
2020/07/13 10:59:20 device HT75H1A00107: battery level 98%, OK
2020/07/13 10:59:21 vm-0: crushing...

Reply:
2020/07/13 10:59:31 saving crash panic:LINE, dump:LINE to /usr/local/google/home/hamzeh/HT75H1A00107/syzkaller/crashes/ea52b31f6e5d74acc123f0a3e7b8d77c021aa5bd
2020/07/13 10:59:31 instances executed: 2, crashes: 2

Reply:
You are right, sorry for creating this confusion. 

I think we might want to lower down restart_time=120 in future as it wont get change to retry as TEST_TIMEOUT=120 as well. so, it will only execute it once.